### ISSUE TITLE
Owntracks iOS app fails to apply configuration on first launch without manual intervention

### ISSUE THREAD

**Issue by Zahrachab** [owntracks/ios#743]
Hi team,

I'm reporting a problem with the Owntracks iOS app (I'm using version 16.4.1). The issue only occurs when I first install (or reinstall) the app and load a new configuration. Subsequent configuration loads do not cause this issue.

The settings are still using the default MQTT settings and the app throws the error `kCFErrorDomainCFNetwork 2 The operation couldn't be completed`. This error can be seen in the screenshot attached.
![1689589295297](https://github.com/owntracks/ios/assets/32749680/5fb70ad8-2284-4012-9009-6c622364b9b1)

To manually fix the problem, I have to go to the settings window, then to the map, then back to the status screen. If I do this, the error disappears and the settings are applied correctly. If I don't, the application remains stuck in error status.
Another manual fix was to manually switch from MQTT to HTTP several times and then change the mode to "move".

Here is my configuration
```

{
  "_type" : "configuration",
  "auth" : true,
  "clientId" : "xxxx",
  "cmd" : true,
  "deviceId" : "",
  "ignoreInaccurateLocations" : 500,
  "locatorDisplacement" : 100,
  "locatorInterval" : 30,
  "locatorPriority" : 2,
  "maxHistory" : 0,
  "mode" : 3,
  "monitoring" : 1,
  "moveModeLocatorInterval" : 30,
  "notificationLocation" : true,
  "password" : "xxxx",
  "positions" : 1,
  "pubTopicBase" : "owntracks/xxxx/client/yyyy",
  "remoteConfiguration" : true,
  "theme" : 0,
  "tid" : "",
  "tls" : true,
  "tlsCaCrt" : "",
  "tlsClientCrt" : "",
  "tlsClientCrtPassword" : "",
  "url" : "https://api.our-domain.com/pub",
  "username" : xxxx"
}
```
I would appreciate any suggestion on how to solve this problem automatically using only a configuration link and without manual intervention in the application.

Thank you for your time.

**Comment by jpmens** [comment]
Could you please elaborate as to _why_ you require this, i.e. why do you load different configurations? What's the use-case?

**Comment by jpmens** [comment]
Are you doing this as a service, i.e. commercially?

**Comment by jpmens** [comment]
Let us know when you're willing to sponsor the feature. Until then I'll mark this as an enhancement request.

**Comment by Zahrachab** [comment]
What's the price point for implementing this please ?

**Comment by ckrey** [comment]
This is a but because the connection was not restarted after the load of configuration via openURL

**Comment by ckrey** [comment]
Available in versions > 16.4.3
